src/figuresClasses.ts

```diff
@@ -1,19 +1,112 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea: ()=> number;
```

Comment:
is your prettier working correctly? 🤔

Reply:
you mean this arrow function type? (if i guess correctly) i ask during lesson and it's a kind of function type that return a number type